proposals/community-strategy/README.md

```diff
@@ -0,0 +1,34 @@
+# Community engagement strategy
+
+The community is the key for a standard. Without adoption, the defined standard is pointless.
+Other standards, such as [Project Connected Home over IP](https://www.connectedhomeip.com/), 
+are moving forward with a lot of support from the industry. 
+
+This directory contains ideas how to push the standard and grow the community behind it.
+
+## 1. Documentation and libraries
+
+Before reaching out to a wide range of developers the base must be present.
+Reference implementations in the major languages, such as C, Javascript and Java, must be present and well maintained.
+An extensive documentation for them must be present as well.
+
+## 2. Reaching out to Open source projects
+
+Integration into other projects is very important. 
+Therefore it makes sense to get in touch with relevant Open source communities.
+In order to help them with their implementations, people must be present on relevant [social media channels](social-media.md).
+Meetups and Conferences are helpful to get in touch with these communities.
+
+## 4. Broader community push
+
+Once there is an adoption in the open source community, it makes sense to push WoT in a broader way.
+This would also include developers which don't contribute to open source projects.
+
+### Conclusion
+
+Since the W3C is a bit slow with its processes, it makes sense to do this outside of the W3C.
+With a seperate community driven website and a small team supported by the major contributor of WoT. 
+[Thingweb](http://www.thingweb.io/) is a good approach, even though the design looks outdated.
+The [node-wot](https://github.com/eclipse/thingweb.node-wot) is missing some important features, such as cbor.
```

Comment:
Node-wot is not perfect, but it is still the reference implementation of [Scripting API](https://w3c.github.io/wot-scripting-api/). We should leverage on it, improving code quality and documentation. I think that a clean architecture and beautiful documentation (like the one in rollup and webpack) will lead to more contributions --> the community will grow. The thing that we miss now is more contributions toward this goal.

Reply:
I am not sure whether CBOR should be there by default. I think this needs to be discussed. Please open an issue on node-wot side if you think so.

Having said that, it is fairly easy to support more content types (see [here](https://github.com/eclipse/thingweb.node-wot/#mediatype-support)). Essentially write a wrapper around a CBOR library and that's it. The downside of having it there by default is that it blows up the bare minimum of node-wot.